core/src/cluster_nodes.rs

```diff
@@ -107,32 +108,42 @@ impl Node {
 
 impl<T> ClusterNodes<T> {
     pub(crate) fn submit_metrics(&self, name: &'static str, now: u64) {
+        let mut epoch_stakes = 0;
         let mut num_nodes_dead = 0;
         let mut num_nodes_staked = 0;
         let mut num_nodes_stale = 0;
+        let mut stake_dead = 0;
+        let mut stake_stale = 0;
```

Comment:
seems like we have enough of these to justify factoring cluster nodes metrics out to their own type (in a future pr)